numba_dppy/compiler.py

```diff
@@ -45,7 +45,7 @@
 )
 
 from . import spirv_generator
-from .dppy_passbuilder import DPPYPassBuilder
+from .dppy_passbuilder import PassBuilder
```

Comment:
`dppy_passbuilder` also contains `dppy`.

Reply:
Yes, I am creating a separate commit to rename everything outside of `numba_dppy.core`.